airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -767,9 +1235,12 @@ definitions:
         type: string
         enum: [InlineSchemaLoader]
       schema:
+        title: Schema
+        description: Describes a streams' schema.
```

Comment:
What do you think about linking to https://docs.airbyte.com/understanding-airbyte/supported-data-types/ ?

Reply:
done